packages/scandipwa/src/route/ProductPage/ProductPage.component.js

```diff
@@ -132,31 +164,87 @@ export class ProductPage extends PureComponent {
         );
     }
 
-    renderAdditionalSections() {
+    renderProductInformationTab() {
         const {
             dataSource,
             parameters,
             areDetailsLoaded
         } = this.props;
 
+        return (
+            <ProductInformation
+              product={ { ...dataSource, parameters } }
+              areDetailsLoaded={ areDetailsLoaded }
+            />
+        );
+    }
+
+    renderProductAttributesTab() {
+        const {
+            dataSource,
+            parameters,
+            areDetailsLoaded
+        } = this.props;
+
+        return (
+            <ProductAttributes
+              product={ { ...dataSource, parameters } }
+              areDetailsLoaded={ areDetailsLoaded }
+            />
+        );
+    }
+
+    renderProductReviewsTab() {
+        const {
+            dataSource,
+            areDetailsLoaded
+        } = this.props;
+
+        return (
+            <ProductReviews
+              product={ dataSource }
+              areDetailsLoaded={ areDetailsLoaded }
+            />
+        );
+    }
+
+    renderProductTabItems() {
+        return Object.values(this.tabMap).reduce((tabRenders, { shouldTabRender, render }) => {
+            if (!shouldTabRender()) {
+                tabRenders.push(render());
+            }
+
+            return tabRenders;
+        }, []);
+    }
```

Comment:
I almost 100% sure this will cause console errors about keys, please add keys to rendered functions, or wrap them into Fragments with keys here.